full-service/src/service/balance.rs

```diff
@@ -175,34 +171,28 @@ where
 
         let conn = self.wallet_db.get_conn()?;
         conn.transaction(|| {
-            let txos = Txo::list_for_address(&address.to_string(), &conn)?;
             let assigned_address = AssignedSubaddress::get(address, &conn)?;
 
-            let mut unspent: u128 = 0;
-            let mut pending: u128 = 0;
-            let mut spent: u128 = 0;
-            let mut secreted: u128 = 0;
-            let mut orphaned: u128 = 0;
-
-            for txo in txos {
-                let status = AccountTxoStatus::get(
-                    &assigned_address.account_id_hex,
-                    &txo.txo.txo_id_hex,
-                    &conn,
-                )?;
-                match status.txo_status.as_str() {
-                    TXO_STATUS_UNSPENT => unspent += txo.txo.value as u128,
-                    TXO_STATUS_PENDING => pending += txo.txo.value as u128,
-                    TXO_STATUS_SPENT => spent += txo.txo.value as u128,
-                    TXO_STATUS_SECRETED => secreted += txo.txo.value as u128,
-                    TXO_STATUS_ORPHANED => orphaned += txo.txo.value as u128,
-                    _ => {
-                        return Err(BalanceServiceError::UnexpectedAccountTxoStatus(
-                            status.txo_status,
-                        ))
-                    }
-                }
-            }
+            let orphaned: u128 = 0;
```

Comment:
So what is actually orphaned? Why is it hardcoded to zero?

Reply:
Orphaned txos have no subaddress assigned, so none of these txos can be orphaned. Added a comment explaining this.